kahuna/public/js/components/gr-crop-image/gr-crop-image.html

```diff
@@ -1,7 +1,9 @@
-<a ng:if="ctrl.canBeCropped" ui:sref="crop({imageId: ctrl.image.data.id})">
-    <gr-icon-label gr-icon="crop">Crop</gr-icon-label>
+<a class="image-info__crop-item image-info__crop-item__button"
```

Comment:
I'd be tempted to avoid putting page specific classes in the base component as it makes it a bit harder to reuse elsewhere as the styling carries through, instead I'd put it [here](https://github.com/guardian/grid/pull/1144/files#diff-845918b04f61694e3ed10f91930a51a3R28).